caravel/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -1,11 +1,56 @@
 import React from 'react';
 import { Panel } from 'react-bootstrap';
+import TimeSeriesLineChart from './charts/TimeSeriesLineChart';
 
-const ChartContainer = function () {
-  return (
-    <Panel header="Chart title">
-      chart goes here
-    </Panel>
-  );
-};
-export default ChartContainer;
+export default class ChartContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      params: this.getParamsFromUrl(),
+      data: props.viz.data,
+      height: window.innerHeight,
+      label1: 'Label 1',
+    };
+  }
+
+  getParamsFromUrl() {
+    const hash = window.location.search;
+    const params = hash.split('?')[1].split('&');
+    const newParams = {};
+    params.forEach((p) => {
+      const value = p.split('=')[1].replace(/\+/g, ' ');
+      const key = p.split('=')[0];
+      newParams[key] = value;
+    });
+    return newParams;
+  }
+
+  formatDates(values) {
+    const newValues = values.map((val) => {
+      return {
+        x: moment(new Date(val.x)).format('MMM D'),
```

Comment:
FYI d3 also has really good date formatting utilities, not sure if it'd be more lightweight than moment.